testing/snes9x/APKBUILD

```diff
@@ -1,16 +1,27 @@
 # Contributor: Taner Tas <taner76@gmail.com>
 # Maintainer: Taner Tas <taner76@gmail.com>
 pkgname=snes9x
-pkgver=1.56.2
-pkgrel=1
+pkgver=1.57
+pkgrel=0
 pkgdesc="Super NES Emulator"
 url="http://www.snes9x.com"
 arch="all"
 license="custom"
 makedepends="
-	nasm autoconf automake libpng-dev zlib-dev libxml2-dev
-	alsa-lib-dev minizip-dev portaudio-dev gtk+3.0-dev sdl-dev
-	libxrandr-dev libepoxy-dev
+	alsa-lib-dev
+	autoconf
+	automake
+	gtk+3.0-dev
+	intltool
+	libpng-dev
+	libxml2-dev
+	libxrandr-dev
+	libepoxy-dev
+	minizip-dev
+	nasm
+	portaudio-dev
+	sdl-dev
+	zlib-dev
 	"
 subpackages="$pkgname-lang $pkgname-doc"
 source="${pkgname}-${pkgver}.tar.gz::https://github.com/${pkgname}git/${pkgname}/archive/${pkgver}.tar.gz"
```

Comment:
please update source using https://github.com/snes9xgit/snes9x instead of substitution - hard to navigate to releases